.github/workflows/openhands-resolver.yml

```diff
@@ -45,6 +45,22 @@ jobs:
         with:
           python-version: "3.12"
 
+      - name: Get latest versions and create requirements.txt
+        run: |
+          python -m pip index versions openhands-resolver > resolver_versions.txt
+          RESOLVER_VERSION=$(head -n 1 resolver_versions.txt | awk '{print $2}' | tr -d '()')
+          echo "openhands-resolver==${RESOLVER_VERSION}" >> requirements.txt
+          cat requirements.txt
+
+      - name: Cache pip dependencies
+        if: github.event.label.name == 'fix-me'
```

Comment:
Overall this is awesome!

Quick question: do you know if this if statement evaluates to true when this workflow is invoked through the external call in `examples/github-resolver.yml`?

Reply:
Yup! Here is an example [workflow](https://github.com/malhotra5/test-repo/actions/runs/11736068442/job/32694720230) log.

I referenced the [workflow on PR externally](https://github.com/malhotra5/test-repo/actions/runs/11736068442/workflow).

Reply:
On second thought, it seems that this shouldn't be the case. Will double check to ensure this is expected behavior.

Reply:
Oh, I definitely would like caching to be enabled when called from an external repo. I was just confirming that this was indeed what happened with the current code.

Reply:
Ah I thought maybe reusable workflows don't share `github.event` between the caller and callee unless they're called in the same repo. The test I provided above had the caller and callee in the same repo so wanted to double check this. Just confirmed that `github.event` is shared between separate repos as well, so we should be good!